### ISSUE TITLE
How to  use Pix3D for training

### ISSUE THREAD

**Issue by XuHHHHH** [hzxie/Pix2Vox#71]
How to use Pix3D dataset for training? I noticed that the training set and validation set are not divided in pix3d.jason.Thanks.

**Comment by hzxie** [comment]
Refer to #20
